### DISCUSSION TITLE
Prebuild next images

### DISCUSSION THREAD

**Discussion by kristianstroka** [vercel/next.js#25186] [Ideas]
### Describe the feature you'd like to request

Is it possible to do prepare next images on build process? So they can be prerendered, specially on static pages, because on live loading, sometimes images loading takes long time to build nextgen images live. 

### Describe the solution you'd like

Prepare images on build process and then just serve them as static files

### Describe alternatives you've considered

Dont know :)

**Comment by kristianstroka** [comment]
@timneutkens Why did you close this feature without any comment on it? :)

**Reply by timneutkens** [reply]
First line of the feature request template:

https://github.com/vercel/next.js/issues/new?assignees=&labels=template%3A+story&template=3.feature_request.yml

> Thanks for taking the time to file a feature request! Please fill out this form as completely as possible.
> Feature requests will be converted to the GitHub Discussions "Ideas" section.

**Comment by max-degterev** [comment]
I am also interested in pre-generating images during build

**Comment by nagman** [comment]
Also interested, as I have a website with a lot of images, regularly updated, and it consumes so much CPU (due to AVIF compression) than it takes 100% of my Pod CPU on Kubernetes

**Reply by stx-chris** [reply]
Hi @nagman I've just opened a fully specified discussion entry. Wanna upvote it to help find a workaround? #46915
